SQL/0000-00-00-schema.sql

```diff
@@ -1522,6 +1522,7 @@ CREATE TABLE `issues` (
   `candID` int(6) DEFAULT NULL,
   `category` varchar(255) DEFAULT NULL,
   `description` longtext DEFAULT NULL,
+  `instrument` varchar(255) DEFAULT NULL,
```

Comment:
I don't think this should be a varchar(255). The `test_names` table should have all instruments and this can/should be a foreign key.

Reply:
@driusan Please check now

Reply:
@driusan My bad, I misunderstood. Please review now.